Makefile

```diff
@@ -15,30 +15,18 @@ ARCH := $(shell uname -m | sed 's/x86_64/amd64/;s/arm64/arm64/;s/aarch64/arm64/'
 GOBASE := $(shell pwd)
 GOBIN := $(GOBASE)/bin
```

Comment:
Well, not related with this PR, but when I tried to run locally the tests, they failed because I lacked to have this directory created. So, add somewhere a `mkdir -p $(GOBIN)` just to ensure it exists.

Reply:
Thanks. Fixed in this on commit 59d51d59c5d3d90099f7da7f89baf92960791ff7 (another PR: #55, starting the orphan deletion code)